config/runtimes/kserve-tritonserver.yaml

```diff
@@ -18,6 +21,8 @@ spec:
     - name: triton
```

Comment:
I guess pytorch is not auto selected because torchserve also support v2 protocol right?

Reply:
yes @yuzisun